Dnn.CommunityForums/config/templates/UserBadge.ascx

```diff
@@ -1,8 +1,11 @@
 <div class="dcf-user-badge-wrap">
-    <div class="dcf-user-badge-image"><span class="dcf-user-badge-count">[BADGE:AWARDCOUNT]</span>
-        <a href="#" class="dcf-user-badge-image-tooltip" data-tooltip="[BADGE:DESCRIPTION]">
-            [BADGE:IMAGEURL:32|<img src="{0}" title="[BADGE:NAME] ([RESX:lastawarded] [BADGE:DATEASSIGNED])" alt="[BADGE:NAME]" data-tooltip="[BADGE:NAME]" />]
-            [BADGE:IMAGEMARKUP|<span title="[BADGE:NAME] ([RESX:lastawarded] [BADGE:DATEASSIGNED])" data-tooltip="[BADGE:NAME]">{0}</span>]
-         </a>
-    </div>
+    <div class="dcf-user-badge-item" data-date="[RESX:lastawarded] [BADGE:DATEASSIGNED]">
```

Comment:
@Timo-Breumelhof does this effectively remove the "awarded" date from the display? Do you think that it should show?

Reply:
> It though it was a bit much?
> 
> I did add it as a data property to show that it exists.
> 
> (more for a someone creating a custom template)

Ok thanks